processor/filterprocessor/config.go

```diff
@@ -55,6 +64,16 @@ type MetricFilters struct {
 
 	// RegexpConfig specifies options for the Regexp match type
 	RegexpConfig *regexp.Config `mapstructure:"regexp"`
+
+	// MetricConditions is a list of OTTL conditions for an ottlmetric context.
+	// If any condition resolves to true, the metric will be dropped.
+	// Supports `and`, `or`, and `()`
+	MetricConditions []string `mapstructure:"metric_conditions"`
```

Comment:
Do we need the "_conditions" suffix? I am not convinced about it. Happy to hear opinions.

Reply:
I added `_conditions` because I wanted to be explicit in the configuration that these were all the OTTL conditions.  In  the yaml I think it looks good 

```yaml
filter/ottl:
  traces:
    span:
      - 'attributes["test"] == "pass"'
    spanevent:
      - 'attributes["test"] == "pass"'
  metrics:
    metric:
      - 'name == "pass"'
    datapoint:
      - 'attributes["test"] == "pass"'
  logs:
    log:
      - 'attributes["test"] == "pass"'
```
but I was worried about new and existing users trying to understand the configuration option.  Adding `_condition` was an attempt to help with that documentation.